Formula/libvirt.rb

```diff
@@ -57,6 +57,7 @@ def install
         -Ddriver_qemu=enabled
         -Ddriver_network=enabled
         -Dinit_script=none
+        -Dqemu_datadir=#{HOMEBREW_PREFIX}/share/qemu
```

Comment:
Why not `Formula["qemu"].pkgshare`?

Reply:
Well, first of all I didn't even know that was a thing O:-)

IIUC that would expand to a path inside QEMU's keg, right? Specifically, right now it would be `/usr/local/Cellar/qemu/6.1.0_1/share/qemu` on x86_64.

What would happen if you later updated QEMU to 6.2.0 without updating/rebuilding libvirt? libvirt would still look for files inside the 6.1.0 directory, while the system would have a 6.2.0 directory now. So it would either not pick up the most recent firmware, or not find it at all (if the 6.1.0 directory is gone after the QEMU update).

Using `HOMEBREW_PREFIX` means libvirt will look at the unversioned directory, which always points to the latest one using symlinks. This feels like the correct approach when loosely coupled pieces of software need to look at each other's files.

My reasoning may very well be wrong though! I'm not overly familiar with the specifics of Homebrew.

Reply:
`Formula["qemu"].pkgshare` will evaluate to `HOMEBREW_PREFIX/"opt/qemu/share/qemu"`, which is a path unaffected by upgrades to `qemu`. It also has the benefit of not getting broken if a user ever happens to do `brew unlink qemu`, which can't be said about `HOMEBREW_PREFIX/"share/qemu"`.

There is some magic in `brew` about the context where `#pkgshare` is evaluated, so if you want to be explicit about it (probably a good idea anyway), you can use `Formula["qemu"].opt_pkgshare`.

Reply:
To clarify, this is a persistent setting: once configured to look into a certain `qemu_datadir` during build, libvirt will always look into that specific directory.

I've looked at a few uses of the `Formula["foo"]` pattern and, from what I can tell, it seems to be mostly used to point the build system to additional include/library paths or running tools like `gtk3-update-icon-cache` during the build. This is a different scenario, because we need to be sure the path specified at build time can be accessed at runtime, even after the QEMU formula/keg is updated.

Reply:
Disregard the second comment, the UI didn't update so I didn't see your reply.

Reply:
Okay that makes sense, thanks for explaining it! I think your suggestion will work.

Just to be sure, the complete option passed to meson should be `-Dqemu_datadir=Formula["qemu"].opt_pkgshare`, right?

Anything else you want me to change in the PR? Or should I make this one change and update it?

Reply:
I think I actually need `#{}` around the expression to get it to evaluate, right? So it should be `-Dqemu_datadir=#{Formula["qemu"].opt_pkgshare}`.

Reply:
Yes.

Reply:
Sorry, to be explicit:

> Anything else you want me to change in the PR? Or should I make this one change and update it?

No, that's all.

> I think I actually need `#{}` around the expression to get it to evaluate, right? 

That's correct.

Reply:
Great! Done now, thank you for the review and the suggestions :)